ui/src/app/views/top-box/component.css

```diff
@@ -22,4 +22,5 @@
   border-left: 1px solid;
   border-right: 1px solid;
   overflow-y: visible;
+  padding: 0 1rem;
```

Comment:
This may be totally unrelated and I'm not sure whether you've fixed it here, but one thing that's been bugging me is that the left and right spacing don't match outside the top box; the left margin is bigger. I think it's actually the padding on the app component.

Reply:
I don't think I have fixed it here, I can take a look and see if I can figure it out.

Reply:
This should fix it now. (Just made the padding on the app-component even. It was in fact that)